docs/faq.md

```diff
@@ -0,0 +1,310 @@
+# Frequently Asked Questions
+
+### ABOUT THE PROJECT
+
+The goal of this project is to make a convenient and user-friendly GUI around Bitcoin Core with a focus on multisignature setup with airgapped (offline) hardware wallets. 
+
+We first wanted to make a new hardware wallet, but after we understood that everything can be hacked, we decided to build a user-friendly Multisig Desktop App and nice DIY Hardware wallet.
+
+**Why is that good for Bitcoin?**
+
+ - User: Better Security with multisig setup 
+ - User: Better Privacy with own node 
+ - HWW Makers: More HW wallets sold 
+ - Node Makers: More Nodes sold 
+ - Network: More nodes running 
+
+We can actually incentivize the Bitcoin community to run their own node with this user-friendly multisig & node setup! 
+
+*Why the name Specter?*
+
+We named it after a vulnerability since we learned that everything can be hacked! Although, when you combine different technical systems in an user-friendly multisig setup you can significantly increase the security of your cold storage. 
+
+
+### GENERAL QUESTIONS
+
+
+*How safe is the app to use? Is it still considered alpha/beta or safe enough to use it with real sats in a Colcard multisig setup?*
+
+It is watch-only (private keys are protected by HWW) and compatible with multisig in Electrum, so even if something breaks you always have a fallback option while we fix the bug. So go for it :)
+
+We try to use default descriptors and derivation paths exactly for this reason - to be compatible with other wallets. Would be nice to keep it this way, but at a certain point we will need to diverge - for example when we add miniscript support.
+
+
+*What does WIP mean?*
+
+WIP means that we don't try to be very backward-compatible at the moment. At some point we may change wallet storage format for example, and you would need to migrate using some script or create wallets from scratch. In this case, we would provide migration scripts.
+
+
+*What's the difference between specter-desktop and specter-DIY?*
+
+Specter-desktop is only a wrapper (watch-only) around Bitcoin Core wallet functionality, mostly, which uses your full node. Bitcoin Core tracks addresses, UTXO (unspent transaction outputs) and composes PSBT (partially-signed bitcoin transactions). But all wallets are watch-only.
```

Comment:
Specter-desktop is a watch-only GUI software wallet running on Bitcoin Core using its wallet and full node functionality. 
Bitcoin Core tracks addresses, UTXO (unspent transaction outputs) and composes PSBT (partially-signed bitcoin transactions).